PrintProxy.py

```diff
@@ -0,0 +1,60 @@
+import re
+import secrets
+from arcpy import GetParameterAsText, SetParameterAsText, AddMessage
+import requests
+
+"""
+Parameters
+input
+0 - ExportWebMapService_URL - URL to the export web map service.
+(e.g. http://localhost/arcgis/rest/services/DEQEnviro/ExportWebMap/GPServer/Export%20Web%20Map)
+1 - Web_Map_as_JSON - Web map JSON txt
+2 - Format - e.g. PDF
+3 - Layout_Template - e.g. Portrait
+output
+4 - Output_File - path to the generated file
+"""
+
+
+def switch_quad_words(json):
+    match = re.search(ur'discover\.agrc\.utah\.gov\/login\/path\/(.*?)\/', json)
+    if match:
+        old_quad = match.groups(1)[0]
+        return json.replace(old_quad, secrets.OPEN_QUAD_WORD)
+    else:
+        return json
+
+
+def make_request(url, webmap, print_format, layout_template):
+    response_json = requests.post(format_url(url), data={
+        'Web_Map_as_JSON': webmap,
+        'Format': print_format,
+        'Layout_Template': layout_template,
+        'f': 'json'
+    }).json()
+    if 'error' in response_json.keys():
+        raise Exception(response_json['error'])
```

Comment:
I think that this depends on the messaging level of the gp service. But yes, the service would return the same error that the print service did.